graphql/mutation_resolver.go

```diff
@@ -191,8 +196,7 @@ func (r *mutationResolver) SaveDistro(ctx context.Context, opts SaveDistroInput)
 	if err = d.ReplaceOne(ctx); err != nil {
 		return nil, InternalServerError.Send(ctx, err.Error())
 	}
-	// TODO EVG-20603: Update with improved distro modified handling
-	event.LogDistroModified(d.Id, usr.Username(), d.NewDistroData())
+	event.LogDistroModified(d.Id, usr.Username(), oldDistro.NewDistroData(), d.NewDistroData())
```

Comment:
nit: would you mind renaming NewDistroData to just DistroData or something? `oldDistro.NewDistroData` seems odd